src/docs/03-concepts/10-http-api.md

```diff
@@ -5,208 +5,1201 @@ permalink: /docs/concepts/http-api
+      "name": "cadence-batcher",
+      "status": "DOMAIN_STATUS_REGISTERED",
+      "description": "Cadence internal system domain",
+      "data": {},
+      "workflowExecutionRetentionPeriod": "604800s",
+      "badBinaries": {
+        "binaries": {}
+      },
+      "historyArchivalStatus": "ARCHIVAL_STATUS_DISABLED",
+      "visibilityArchivalStatus": "ARCHIVAL_STATUS_DISABLED",
+      "activeClusterName": "cluster0",
+      "clusters": [
+        {
+          "clusterName": "cluster0"
+        }
+      ],
+      "failoverVersion": "-24",
+      "isolationGroups": {}
+    },
+    {
+      "id": "59c51119-1b41-4a28-986d-d6e377716f82",
+      "name": "cadence-shadower",
+      "status": "DOMAIN_STATUS_REGISTERED",
+      "description": "Cadence internal system domain",
+      "data": {},
+      "workflowExecutionRetentionPeriod": "604800s",
+      "badBinaries": {
+        "binaries": {}
+      },
+      "historyArchivalStatus": "ARCHIVAL_STATUS_DISABLED",
+      "visibilityArchivalStatus": "ARCHIVAL_STATUS_DISABLED",
+      "activeClusterName": "cluster0",
+      "clusters": [
+        {
+          "clusterName": "cluster0"
+        }
+      ],
+      "failoverVersion": "-24",
+      "isolationGroups": {}
+    },
+    {
+      "id": "32049b68-7872-4094-8e63-d0dd59896a83",
+      "name": "cadence-system",
+      "status": "DOMAIN_STATUS_REGISTERED",
+      "description": "cadence system workflow domain",
+      "ownerEmail": "cadence-dev-group@uber.com",
+      "data": {},
+      "workflowExecutionRetentionPeriod": "259200s",
+      "badBinaries": {
+        "binaries": {}
+      },
+      "historyArchivalStatus": "ARCHIVAL_STATUS_DISABLED",
+      "visibilityArchivalStatus": "ARCHIVAL_STATUS_DISABLED",
+      "activeClusterName": "cluster0",
+      "clusters": [
+        {
+          "clusterName": "cluster0"
+        }
+      ],
+      "failoverVersion": "-24",
+      "isolationGroups": {}
+    },
+    {
+      "id": "d7aff879-f524-43a8-b340-5a223a69d75b",
+      "name": "sample-domain",
+      "status": "DOMAIN_STATUS_REGISTERED",
+      "data": {},
+      "workflowExecutionRetentionPeriod": "259200s",
+      "badBinaries": {
+        "binaries": {}
+      },
+      "historyArchivalStatus": "ARCHIVAL_STATUS_ENABLED",
+      "historyArchivalUri": "file:///tmp/cadence_archival/development",
+      "visibilityArchivalStatus": "ARCHIVAL_STATUS_ENABLED",
+      "visibilityArchivalUri": "file:///tmp/cadence_vis_archival/development",
+      "activeClusterName": "cluster0",
+      "clusters": [
+        {
+          "clusterName": "cluster0"
+        }
+      ],
+      "isGlobalDomain": true,
+      "isolationGroups": {}
+    }
+  ],
+  "nextPageToken": ""
+}
+```
+
+</details>
+
+------------------------------------------------------------------------------------------
+
+### Meta API
+
+------------------------------------------------------------------------------------------
+
+### Visibility API
```

Comment:
If you are publishing this without any content initially, please write:
    **Coming Soon...**